backends/graphics/opendingux/opendingux-graphics.cpp

```diff
@@ -22,13 +22,16 @@
 #include "backends/graphics/opendingux/opendingux-graphics.h"
 
 void OpenDinguxGraphicsManager::initGraphicsSurface() {
-	Uint32 flags = _videoMode.isHwPalette ? (SDL_HWSURFACE | SDL_HWPALETTE | SDL_DOUBLEBUF) : SDL_SWSURFACE;
```

Comment:
Could it be that RS90 does not have `HwPalette` in some mode?

Reply:
Hardware palettes were already enforced on RS90 in the overridden setupHardwareSize() function.